packages/ffe-cards-react/src/index.d.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+export interface CardBaseProps extends React.HTMLAttributes<HTMLDivElement> {
+    children: React.ReactNode;
+    className?: string;
+    element?: React.ReactNode;
```

Comment:
Right, a node can also be a number or anything else that is renderable but we don't want that here.

Reply:
There's another problem here though. I want my cards to be links so I need to pass inn the `Link` component from react-router (or reach-router) and THAT component again expects a `to` prop which is used in place of `href`.

How do you normally express this in TS? If element is of the `Link` instance from a given library we must be able to pass the `to` prop, but otherwise it makes no sense to do so.